integ-test/src/test/java/org/opensearch/sql/legacy/PluginIT.java

```diff
@@ -70,8 +70,12 @@ public void sqlDeleteSettingsTest() throws IOException {
             "{\n"
                 + "  \"error\": {\n"
                 + "    \"reason\": \"Invalid SQL query\",\n"
-                + "    \"details\": \"DELETE clause is disabled by default and will be deprecated. Using "
-                + "the plugins.sql.delete.enabled setting to enable it\",\n"
+                + "    \"details\": \""
```

Comment:
comparing old one, the message is more confuse for customer.

Reply:
https://github.com/opensearch-project/sql/pull/839#discussion_r981653387